astester.imports.js

```diff
@@ -0,0 +1,39 @@
+export default function (memory) {
+  const Storage = new Map();
+  return {
+    env: {
+      memory,
+      abort(msg, file, line, col) {
+      getString(msg)
+
+       console.log("error", text)
+      },
+      log(ptr) {
+       console.log("log", text)
+      },
+  },
+  massa: {
+      memory,
+      assembly_script_generate_event(string) {
+       console.log("event", string)
```

Comment:
not sure you can do that. I think you will have a pointer. You should have a look at what Max did for the playground.

Reply:
indeed, problem solved